### ISSUE TITLE
handleOpenUrl not called with cold start on IOS 

### ISSUE THREAD

**Issue by henkkelder** [EddyVerbruggen/Custom-URL-scheme#210]
Like many others I am struggling to get handleOpenURL called on IOS when the app is not loaded (cold start). I am using Phonegap Build using cli-6.3.0 on IOS 10.

I am certain it did work 2 years ago in another app I made, but now if does not work in two different apps.

Things I've tried:

- Moved handleOpenURL to a <script> block inside the body of my single page HTML file;
- Added timeouts up to 15 seconds before calling the actual handler;
- Modifying CSP: "... default-src * opwtb2015:; ..." (the latter is my custom scheme)
- Removing CSP al together;
- Added -plist-info items.

I've seen solutions where folks modified code in the plugin. That is not a option for me since I am using PGB.

Basically, I wonder: Does this still work, or in other words should handleOpenURL be called on cold start?

**Comment by henkkelder** [comment]
Ooops.. Was my own mistake. I configured my app to launch 'start.html' which in turn redirected to my real 'index.html'. Since my handleOpenURL was in 'index.html' this was simply not loaded when the handler was called during cold start. 

Solved.
